### PR TITLE
add model lesson, result, answer and show wordlist

### PR DESCRIPTION
**Pull request by thanhbk55**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -10,3 +10,9 @@
   description = Faker::Lorem.sentence(5)
   Category.create!(name: name, description: description) 
 end
+
+categories = Category.order(:created_at).take(5)
+20.times do
+  content = Faker::Lorem.sentence(5)
+  categories.each { |category| category.words.create!(content: content) }
```
Comment (phamkykhoi) [inline db/seeds.rb]: Remove space  at  { |category|
20 move to Constant or Settings

File: app/views/categories/index.html.erb [hunk 2]
```diff
@@ -5,4 +5,17 @@
...
+</div>
+
+<% @categories.each do |category| %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title"><%= category.name %></h3>
+    </div>
+   <div class="panel-body">
+       <div class="categories-word-<%= category.id %>">
+        	<%= render category.words %>
```
Comment (phamkykhoi) [inline app/views/categories/index.html.erb]: Indent

File: config/routes.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@
   get "login" => "sessions#new"
   post "login" => "sessions#create"
   delete "logout" => "sessions#destroy"
+  get 'category' => "categories#index"
```
Comment (vietleminh) [inline config/routes.rb]: ' ' => " "

File: app/views/words/_word.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<li id="word-<%= word.id %>">
+   <span class="word"><%= word.content %></span>
+</li>
```
Comment (vietleminh) [inline app/views/words/_word.html.erb]: new line

File: app/views/categories/index.html.erb [hunk 5]
```diff
@@ -5,4 +5,17 @@
...
+        </div>
+    </div>
+</div>
+<% end %>
```
Comment (vietleminh) [inline app/views/categories/index.html.erb]: new line

File: app/controllers/categories_controller.rb [hunk 6]
```diff
@@ -1,5 +1,6 @@
 class CategoriesController < ApplicationController
   def index
     @categories = Category.all.paginate page: params[:page], per_page: 5
```
Comment (phamkykhoi) [inline app/controllers/categories_controller.rb]: Remove all
Add include ( Read more Solution to N + 1 queries problem )

File: app/controllers/categories_controller.rb [hunk 7]
```diff
@@ -1,5 +1,6 @@
...
+    @words = Word.all.paginate page: params[:page], per_page: 5
```
Comment (phamkykhoi) [inline app/controllers/categories_controller.rb]: Move 5 to Settings

Comment (phuongbk) [inline app/controllers/categories_controller.rb]: No need!

**Pull request closed without merging**